operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -530,9 +530,6 @@ distributor:
 
```

Comment:
Ruler, alertmanager stateless deployment mode also has:
```
  strategy:
    type: RollingUpdate
    rollingUpdate:
      maxSurge: 0
      maxUnavailable: 1
```
Let's remove that as well.

Also query-scheduler, but we only support two at the moment, so that's probably fine.

Reply:
Thank you for the explanation.